src/ICeWall/doc/set_bump_options.md

```diff
@@ -0,0 +1,60 @@
+## set_bump_options
+
+### Synopsis
+```
+  % set_bump_options \
+        [-pitch pitch] \
+        [-spacing_to_edge spacing] \
+        [-bump_pin_name pin_name] \
+        [-cell_name bump_cell_table] \
+        [-num_pads_per_tile value] \
```

Comment:
It looks like the max value the router you wrote can handle is 5, so you should probably check the value to make sure its greater than 0 and less than= 5.